cob_twist_controller/src/inverse_jacobian_calculations/inverse_jacobian_calculation.cpp

```diff
@@ -79,15 +80,15 @@ Eigen::MatrixXd PInvBySVD::calculate(const TwistControllerParams& params,
             singularValuesInv(i) = singularValues(i) / (pow(singularValues(i), 2) + pow(params.beta, 2));
         }
         // Formula 20 - additional part - numerical filtering for least singular value m
-        uint32_t m = singularValues.rows();
-        singularValuesInv(m) = singularValues(m) / (pow(singularValues(m), 2) + pow(params.beta, 2) + pow(lambda, 2));
+        uint32_t m = singularValues.rows()-1;
```

Comment:
Previously, ```m``` has been computed like
```uint32_t m = singularValues.rows();```
now it is like
```uint32_t m = singularValues.rows()-1;```
:point_right: remove the ```-1```